spec/fixtures/wp/335/yr/5649/cocina.json

```diff
@@ -0,0 +1 @@
:3,"hasMimeType":"text/plain","hasMessageDigests":[{"type":"sha1","digest":"0bd6aa284eb75096d74653ca38213bff2cd553a4"},{"type":"md5","digest":"55b1da0dba98cf8a81dc3f852e11d0bb"}],"access":{"view":"world","download":"world","controlledDigitalLending":false},"administrative":{"publish":true,"sdrPreserve":true,"shelve":true}}]}},{"type":"https://cocina.sul.stanford.edu/models/resources/file","externalIdentifier":"https://cocina.sul.stanford.edu/fileSet/wp335yr5649-wp335yr5649_2","label":"Example of accessing the SQLite database from Matlab","version":3,"structural":{"contains":[{"type":"https://cocina.sul.stanford.edu/models/file","externalIdentifier":"https://cocina.sul.stanford.edu/file/wp335yr5649-wp335yr5649_2/example_using_DPT_observed.m","label":"example_using_DPT_observed.m","filename":"example_using_DPT_observed.m","size":3979,"version":3,"hasMimeType":"text/plain","hasMessageDigests":[{"type":"sha1","digest":"adb8629ffdd307153bca3c8f568f999078c14ea2"},{"type":"md5","digest":"dd3b78e9979da590effa82bc5fe7d17f"}],"access":{"view":"world","download":"world","controlledDigitalLending":false},"administrative":{"publish":true,"sdrPreserve":true,"shelve":true}}]}},{"type":"https://cocina.sul.stanford.edu/models/resources/file","externalIdentifier":"https://cocina.sul.stanford.edu/fileSet/wp335yr5649-wp335yr5649_3","label":"Raw observation dataset as a text file in CSV format","version":3,"structural":{"contains":[{"type":"https://cocina.sul.stanford.edu/models/file","externalIdentifier":"https://cocina.sul.stanford.edu/file/wp335yr5649-wp335yr5649_3/DPT_observed.csv","label":"DPT_observed.csv","filename":"DPT_observed.csv","size":13562472,"version":3,"hasMimeType":"text/plain","hasMessageDigests":[{"type":"sha1","digest":"f0a027b1a6836e37321656b1b772955ed55e3ff0"},{"type":"md5","digest":"b97b7fa3124d6d2785d58e4c825320c3"}],"access":{"view":"world","download":"world","controlledDigitalLending":false},"administrative":{"publish":true,"sdrPreserve":true,"shelve":true}}]}},{"type":"https://cocina.sul.stanford.edu/models/resources/file","externalIdentifier":"https://cocina.sul.stanford.edu/fileSet/wp335yr5649-wp335yr5649_4","label":"Raw observation dataset as an SQLite3 database","version":3,"structural":{"contains":[{"type":"https://cocina.sul.stanford.edu/models/file","externalIdentifier":"https://cocina.sul.stanford.edu/file/wp335yr5649-wp335yr5649_4/DPT_observed.db","label":"DPT_observed.db","filename":"DPT_observed.db","size":12660736,"version":3,"hasMimeType":"application/octet-stream","hasMessageDigests":[{"type":"sha1","digest":"5266ad763a8d7a09e717c60851f32346181f52c1"},{"type":"md5","digest":"db778406364b853caae11c80fa192987"}],"access":{"view":"world","download":"world","controlledDigitalLending":false},"administrative":{"publish":true,"sdrPreserve":true,"shelve":true}}]}}],"hasMemberOrders":[],"isMemberOf":["druid:md919gh6774"]},"created":"2012-12-13T17:45:43.000+00:00","modified":"2022-09-28T20:07:06.000+00:00","lock":"druid:wp335yr5649=0"}
```

Comment:
Please use the existing pattern for creating test objects instead of adding fixtures? (See, e.g., versioned_files_helper).